acq4/drivers/pvcam/pvcam.py

```diff
@@ -60,9 +60,16 @@
         ('binningX', 1, [1,2,3,4,8,12,16,24,32,48,64,128,256,512]),
         ('binningY', 1, [1,2,3,4,8,12,16,24,32,48,64,128,256,512]),
     ],
+
+    'RetigaR1': [
+        ('READOUT_PORT', 0),  ## Fastest port for R1
+        ('SPDTAB_INDEX', 2),  ## Fastest option for Q57
```

Comment:
These settings have been in use for the past 4 months or so without problems. I propose that we keep the settings since they're working but remove both comments since we don't know that they are true.